src/cpp/desktop/DesktopMain.cpp

```diff
@@ -308,7 +308,6 @@ int main(int argc, char* argv[])
       bool devMode = false;
 
       // check for debug configuration
-#ifndef NDEBUG
```

Comment:
This should instead be something like `#ifndef RSTUDIO_PACKAGE_BUILD` (not sure if we populate that define or something similar already)